core/arch/arm/kernel/virtualization.c

```diff
@@ -169,23 +169,23 @@ void virt_init_memory(struct tee_mmap_region *memory_map)
 }
 
 
-static int configure_guest_prtn_mem(struct guest_partition *prtn)
+static TEE_Result configure_guest_prtn_mem(struct guest_partition *prtn)
 {
-	int ret;
+	TEE_Result res;
```

Comment:
= TEE_xxx;  note the function either succeeds or returns OUT_OF_MEMORY.

Reply:
I'll fix.